drivers/st77xx/Kconfig

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menuconfig MODULE_ST77XX
+    bool "ST77xx display driver"
```

Comment:
I think, in order to handle default selection, we should use a pattern like saul_default.  See si114x as an example.